package.json

```diff
@@ -59,7 +59,7 @@
     "release": "gulp release",
     "federalist": "npm run build && npm run build:storybook",
     "start": "npm run start:storybook",
-    "test": "snyk test && npm run lint && gulp typecheck && gulp lintSass && gulp test",
+    "test": "snyk test && npm run lint && gulp buildUSWDS && gulp typecheck && gulp lintSass && gulp test",
```

Comment:
Added `buildUSWDS` after encountering errors (for more than just usa-icon) when no dist directories existed. 

I was conflicted on adding it, so let me know if you would like to see this removed.